metricbeat/module/elasticsearch/node_stats/_meta/fields.yml

```diff
@@ -1,158 +1,167 @@
-- name: node.stats
+- name: node
   type: group
   description: >
-    node_stats
+    Node
   release: beta
   fields:
-    - name: indices
-      type: group
+    - name: id
```

Comment:
I would assume this is now a duplicated field as it's also in `node` metricset? I general I would recommend to configure "shared" fields on the module level.